.teamcity/_self/projects/WebApp.kt

```diff
@@ -55,6 +56,15 @@ object BuildDockerImage : BuildType({
 			checked = "true",
 			unchecked = "false"
 		)
+		checkbox(
+			name = "UPDATE_BASE_IMAGE_CACHE",
+			value = "false",
+			label = "Update the base image from the cache.",
+			description = "Updates the base image by copying .cache files from the current build. Runs on trunk by default if the cache invalidates during the build.",
+			checked = "true",
+			unchecked = "false"
+		)
+		param("env.WEBPACK_CACHE_INVALIDATED", "false")
```

Comment:
This flag is ultimately set by a service message. I added a checkbox above so that we can also trigger it on branches.